packages/config/config/devices/0x0346/keypad.json

```diff
@@ -0,0 +1,387 @@
+// Ring Keypad
+// Param information taken from https://support.ring.com/hc/en-us/article_attachments/360051851012/Keypad_Zwave_UK.pdf
+{
+	"manufacturer": "Ring",
+	"manufacturerId": "0x0346",
+	"label": "Keypad",
+	"description": "Keypad",
+	"devices": [
+		{
+			"productType": "0x0101",
+			"productId": "0x0201"
+		},
+		{
+			"productType": "0x0101",
+			"productId": "0x0202"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"1": {
+			"label": "Battery Report Interval",
+			"unit": "min",
+			"valueSize": 2,
+			"minValue": 70,
+			"maxValue": 1440,
+			"defaultValue": 70,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Z-Wave Sleep Timeout",
```

Comment:
What does this mean? What does it do? After this it's asleep? How long it stays awake? Please make it more description. I'd say Wakeup Duration or something that's clearer.